jsapp/js/components/account/accountSidebar.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import {RouteComponentProps} from 'react-router'
+import {ROUTES} from 'js/router/routerConstants'
+import bem from 'js/bem'
+import LoadingSpinner from 'js/components/common/loadingSpinner'
+import Icon from 'js/components/common/icon'
+import './accountSidebar.scss'
+
+const ACCOUNT_SETTINGS_HREF = '#' + ROUTES.ACCOUNT_SETTINGS
+const DATA_STORAGE_HREF = '#' + ROUTES.DATA_STORAGE
+const SECURITY_HREF = '#' + ROUTES.SECURITY
+
+type AccountSidebarProps = RouteComponentProps<
+  {
+    submissionsPerMonth: number
+    /* TODO: Placeholder from mockups, naming and typing subject to change
+      dataStoreage: any,
+      transcriptionMinutes: any,
+	    machineTranslation: any,
+	 */
+  },
+  {}
+>
+
+type AccountSidebarState = {
+	isLoading: boolean
+}
+
+export default class AccountSidebar extends React.Component<
+  AccountSidebarProps,
+  AccountSidebarState
+> {
+  constructor(props: AccountSidebarProps) {
+    super(props)
+    this.state = {
+      isLoading: true,
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+      isLoading: false,
+    })
+  }
+
+  isAccountSelected(): boolean {
+    return (
+      location.hash.split('#')[1] === ROUTES.ACCOUNT_SETTINGS
+    )
+  }
+
+  isDataStorageSelected(): boolean {
+    return location.hash.split('#')[1] === ROUTES.DATA_STORAGE
+  }
+
+  isSecuritySelected(): boolean {
+    return location.hash.split('#')[1] === ROUTES.SECURITY
+  }
+
+  render() {
+    let sidebarModifier: string = 'account'
```

Comment:
You can just write `let sidebarModifier = 'account'`, as TypeScript would extrapolate the type from the initial string value